prometheus_aws_guardduty_exporter/collector.py

```diff
@@ -22,53 +23,51 @@ def collect(self):
             "The total number of scrape errors",
             labels=["region"])
 
-        # Customize the boto client config
-        botoConfig = botocore.client.Config(connect_timeout=2, read_timeout=10, retries={"max_attempts": 2})
-
-        for region in self.regions:
-            try:
-                # Scrape GuardDuty statistics from the region
-                regionStats = self._collectMetricsByRegion(region, botoConfig)
-
-                # Add the findings to the exporterd metric
-                for severity, count in regionStats.items():
-                    currentFindingsMetric.add_metric(value=count, labels=[region, severity])
-            except Exception as error:
-                logging.getLogger().error(f"Unable to scrape GuardDuty statistics from {region} because of error: {str(error)}")
-
-                # Increase the errors count
-                self.scrapeErrors[region] += 1
-
-        # Update the scrape errors metric
-        for region, errorsCount in self.scrapeErrors.items():
-            scrapeErrorsMetric.add_metric(value=errorsCount, labels=[region])
+        pool = Pool(len(self.regions))
```

Comment:
Is there a good reason to instance a new pool for each `collect()` invocation? Shouldn't this be created in the constructor?